doc-src/SASS_REFERENCE.md

```diff
@@ -676,6 +676,7 @@ SassScript supports seven main data types:
 * nulls (e.g. `null`)
 * lists of values, separated by spaces or commas (e.g. `1.5em 1em 0 2em`, `Helvetica, Arial, sans-serif`)
 * maps from one value to another (e.g. `(key1: value1, key2: value2)`)
+* callable values that refer to functions
```

Comment:
I'd just describe these as "first-class functions", at least until we have first-class mixins as well. That's more consistent with the terminology from other languages.